src/components/MapView/PendingMapView.tsx

```diff
@@ -16,13 +16,15 @@ function PendingMapView({title = '', subtitle = '', style}: PendingMapViewProps)
             {hasTextContent ? (
                 <BlockingView
                     icon={Expensicons.EmptyStateRoutePending}
+                    iconColor={undefined}
                     title={title}
                     subtitle={subtitle}
                     shouldShowLink={false}
                 />
             ) : (
                 <View style={[styles.flex1, styles.alignItemsCenter, styles.justifyContentCenter, styles.ph10]}>
                     <Icon
+                        fill={undefined}
```

Comment:
Pointed out previously. Another case where we can avoid repeatedly setting a prop to undefined when using the component.